src/crypto/tls/handshake_server_tls13.go

```diff
@@ -684,25 +706,29 @@ func illegalClientHelloChange(ch, ch1 *clientHelloMsg) bool {
 func (hs *serverHandshakeStateTLS13) sendServerParameters() error {
 	c := hs.c
 
-	earlySecret := hs.earlySecret
-	if earlySecret == nil {
-		earlySecret = hs.suite.extract(nil, nil)
-	}
-	hs.handshakeSecret = hs.suite.extract(hs.sharedKey,
-		hs.suite.deriveSecret(earlySecret, "derived", nil))
-
 	// Confirm ECH acceptance.
-	if hs.clientHello.echIsInner {
-		confTranscript := cloneHash(hs.transcript, hs.suite.hash)
-		if confTranscript == nil {
+	if hs.echIsInner() {
+		// Clear the last 8 bytes of the ServerHello.random in preparation for
+		// computing the confirmation hint.
```

Comment:
```suggestion
		// computing the confirmation signal.
```

Reply:
I removed the comment because it's not actually relevant here. (It was copy-pasted from the SH confirmation signal computation.)